modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/interface/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import AJAX from '../AJAX/index';
+import * as Events from '../eventsDefinitions';
+import * as FormUtils from '../forms/index';
+import * as BaseUtils from '../index';
+
+export default {
+	AJAX,
+	BaseUtils,
+	FormUtils,
+	Events
+};
```

Comment:
@marco-leo @FabioDiegoMastrorilli @garofalog

Exposed utils. These are widely used in JSP's. Could you please evaluate the list as said above?